README.md

```diff
@@ -13,6 +13,8 @@ Web application for Pliplox
 ### Working on the project
 
 * In the project directory: `>_ cd PROJECT_NAME`
+* Install node dependencies: `npm i`
+* Install yarn: `npm i yarn`
```

Comment:
wat?

Reply:
cambiar esto por "You must have yarn installed", pq para cada OS es distinto y el install dependencies debe ser como:
 
```bash
yarn install
```